main.go

```diff
@@ -461,23 +428,35 @@ func main() {
 	joinedAPKOutputPaths := strings.Join(signedAPKPaths, "|")
 	joinedAABOutputPaths := strings.Join(signedAABPaths, "|")
 
-	if err := tools.ExportEnvironmentWithEnvman("BITRISE_SIGNED_APK_PATH", joinedAPKOutputPaths); err != nil {
-		log.Warnf("Failed to export Build Artifact, error: %s", err)
-	}
-	log.Donef("The Signed Build Artifact path is now available in the Environment Variable: BITRISE_SIGNED_APK_PATH (value: %s)", joinedAPKOutputPaths)
+	// APK
+	if len(signedAPKPaths) > 0 {
+		if err := tools.ExportEnvironmentWithEnvman("BITRISE_SIGNED_APK_PATH", signedAPKPaths[len(signedAPKPaths)-1]); err != nil {
+			log.Warnf("Failed to export Build Artifact, error: %s", err)
```

Comment:
Please add the path to the log, it would make debuging more easier

Reply:
Fixed